src/pages/index.js

```diff
@@ -62,36 +104,106 @@ function handleImageClick(cardData) {
 
 //edit profile
 function handleProfileSubmit(data) {
-  userInfo.setUserInfo({name: data.name, job: data.description});
-editPopup.close(); }
-
-//add card
+  console.log(data);
+  api.updateUserInfo({
+    name: data.name,
+    about: data.about
+  })
+    .then(updatedUserData => {
+      user.setUserInfo({
+        name: updatedUserData.name, 
+        about: updatedUserData.about || updatedUserData.dascription || updatedUserData.bio
```

Comment:
`updatedUserData` is a ready object. The sever can't send `dascription` or `bio` in it


```js
    .then(updatedUserData => {
      user.setUserInfo(updatedUserData);
```